safe/impact_functions/bases/calculation_result.py

```diff
@@ -0,0 +1,95 @@
+# coding=utf-8
+from safe.common.exceptions import NoImpactClassFoundError
+
+__author__ = 'Rizky Maulana Nugraha "lucernae" <lana.pcfre@gmail.com>'
+__date__ = '07/05/15'
+
+
+class VectorImpactCalculation(object):
+    """Class to store impact function calculation results
+    """
+
+    def __init__(self):
+        self._impact_map = None
+        self._hazard_map = None
+        self._impacted_count = None
+        self._total_count = None
+        self._impact_layer = None
+
+    @property
+    def hazard_map(self):
+        return self._hazard_map
+
+    @hazard_map.setter
+    def hazard_map(self, value):
+        """The classified hazard map by hazard type. Only valid for
```

Comment:
First line of docstring should be single line followed by a full stop and a new line.